Clarify instantly that Docker Desktop for Linux uses its own context (#16105)
docker/docs: desktop/install/linux-install.md

> employees OR more than $10 million USD in annual revenue) requires a paid
> subscription.

>Important
>

> >Be aware that Docker Desktop on Linux runs a Virtual Machine (VM), creates and uses a custom docker context `desktop-linux` on startup. 

Reviewer: ```suggestion
>Docker Desktop on Linux runs a Virtual Machine (VM) so creates and uses a custom docker context `desktop-linux` on startup. 
```
